### PR TITLE
Fixed associations between coffee and coffee shops and modified _id t…

### PR DESCRIPTION
**Pull request by jmsteen**
FIxed associations. Changed _id to id.

### CODE REVIEW

File: server/models/CoffeeShop.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const CoffeeShopSchema = new Schema({
     required: true
   },
   founded: {
-    type: Date,
+    type: Number,
```
Comment (hegelocampus) [inline server/models/CoffeeShop.js]: I'm a little wary about storing a date as a number because it could possibly cause issues when trying to interact with that object in the front end when we're normally expecting strings, but this is fine if its the best way to do it.

File: server/schema/types/root_query_type.js [hunk 2]
```diff
@@ -34,23 +34,23 @@ const RootQueryType = new GraphQLObjectType({
             }
         },
         coffee: {
-            type: CoffeeType,
+            type: require("./coffee_type"),
```
Comment (hegelocampus) [inline server/schema/types/root_query_type.js]: Why did you change it back to this? Was it not working again? I think the previous syntax looked much cleaner.

**Review by hegelocampus — COMMENTED** [review]
There are a couple things here I don't quite understand and I'd appreciate a little bit of clarification before we pull it in.

File: server/schema/types/root_query_type.js (hunk 2 above)
Comment (jmsteen) [reply server/schema/types/root_query_type.js]: It was part of the debugging process for errors I was getting with the coffee association. However, changing it back still works so I agree that it's cleaner to revert back

File: server/models/CoffeeShop.js (hunk 1 above)
Comment (jmsteen) [reply server/models/CoffeeShop.js]: I'm ok with making it a string. I changed it to a Number b/c it had to agree with the GraphQLInt type. But that makes sense regarding the frontend, so I'll make it a string.

Comment (uwgnol1612) [reply server/models/CoffeeShop.js]: If all we do is just displaying the year then I think string is fine

Comment (jmsteen) [reply server/models/CoffeeShop.js]: Yep agreed! I fixed this issue in the push I just made.

File: server/schema/types/root_query_type.js (hunk 2 above)
Comment (jmsteen) [reply server/schema/types/root_query_type.js]: @hegelocampus @uwgnol1612  Made both of these changes and ready for your review.

**Review by uwgnol1612 — APPROVED** [review]
